docs/diagrams/TagSequenceDiagram.puml

```diff
@@ -0,0 +1,57 @@
+@startuml
+!include style.puml
+
+autonumber
```

Comment:
Is the numbering necessary? I think the other diagrams don't have it

Reply:
I'll keep it consistent and remove the numbering then